### PR TITLE
Fixes #8770: Correct error message when SHA512 algorithm is not available

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/8770

### CODE REVIEW

File: src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala [hunk 1]
```diff
@@ -231,7 +231,7 @@ object AixPasswordHashAlgo extends Loggable {
       // was introduced in Java 8.
       // Fallback to ssha1 hash.
       logger.error("Your current Java installation does not support PBKDF2WithHmacSHA512 algorithm, " +
-          "which is necessary for {ssha256} hash. Falling back to {ssha1} hashing scheme")
+          "which is necessary for {ssha512} hash. Falling back to {ssha1} hashing scheme")
```
Comment (jooooooon) [inline src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala]: Looks good to me for the wording, but shouldn't we fall back to ssha256 first? (which would of course fall back to ssha1 in turn if not available)

Comment (jooooooon) [inline src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala]: @VinceMacBuche ?

Comment (VinceMacBuche) [inline src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala]: If ssha512 is not supported, ssha256 is not supported too (it's written a few lines above)

Comment (jooooooon) [inline src/main/scala/com/normation/cfclerk/domain/AixPasswordHashAlgo.scala]: OK, then I guess we can merge this.

**Pull request merged**